app/assets/v2/js/pages/profile-trust.js

```diff
@@ -657,6 +657,104 @@ Vue.component('sms-verify-modal', {
   }
 });
 
+Vue.component('ens-verify-modal', {
+  data: function() {
+    return {
+      showValidation: false,
+      static_url: document.contxt.STATIC_URL,
+      validationStep: 0,
+      validationError: '',
+      validationErrorMsg: '',
+      validated: false
+    };
+  },
+  mounted: function() {
+    $(document).on('click', '#verify-ens-link', function(event) {
+      event.preventDefault();
+      this.showValidation = true;
+    }.bind(this));
+    this.checkENSValidation();
+  },
+  template: `
+    <b-modal id="ens-modal" @hide="dismissVerification()" :visible="showValidation" center hide-header hide-footer>
+      <template v-slot:default="{ hide }">
+        <div class="mx-5 mt-5 mb-4 text-center">
+          <div class="mb-3">
+            <h1 class="font-bigger-4 font-weight-bold">Verify with ENS</h1>
+            <p>
+              ENS is a name service built on Ethereum. It offers a secure and decentralized way to address resources using human-readable names.
+              <p class="mb-4">
+                <a href="https://ens.domains/about">Learn more.</a>
+              </p>
+            </p>
+          <div>
+            <h2 class="font-bigger-4 font-weight-bold">You need to have these requirements:</h2>
+            <ol>
+              <li>You have a preferred payout address in <a href="/settings/account">settings</a>. <span :class="\`mr-2 fas fa-\${ validationStep > 1 ? 'check check gc-text-green' : 'times gc-text-pink' }\`"></span></li>
+              <li>Your preferred payout address has an ENS domain associated, you can get one <a href="https://app.ens.domains/">here</a>. <span :class="\`mr-2 fas fa-\${ validationStep > 2 ? 'check gc-text-green' : 'times gc-text-pink' }\`"></span></li>
```

Comment:
I though reuse the address that users are already registered in the gitcoin platform (maybe is not the best name for the address) instead of requiring an address and make the verification fast, another option is add one text field populating this with the address registered and allow the users change it if its required. :thinking:

Reply:
I think we should be consistent with other verification mechanism. If others (BrightId, Twitter, POAP, etc) is using the payout address, that's fine by me. @owocki can you verify if that's the case?

Reply:
> another option is add one text field populating this with the address registered and allow the users change it if its required

this seems like the most flexible option to me.  but you could prefill it with the prefered payout addr

Reply:
i added the text field

Reply:
💯